{{cookiecutter.project_name}}/.gitignore

```diff
@@ -3,6 +3,13 @@
 *~
 buck-out/
 
+{% if not cookiecutter.ide_integration -%}
+# IDEs
+/.idea/
```

Comment:
This is a very technical detail that is not important but there are a few files in the `.idea` folder that Jetbrains thinks are fine to commit. But let's ignore this.

Reply:
yeah, that’s what I mean with

> Maybe the variable should be a multiple choice for “none”, “vs code”, “all”, with the ability to extend support for PyCharm and so on?